src/dashboard/components/Dashboard/Catalog/Categories/Editor/CategoryEditorWrapper.tsx

```diff
@@ -262,7 +263,9 @@ const CategoryEditorWrapper = ({
           />
         </Grid>
       </Grid>
-      {creatingNew ? null : <DeleteEntityButton onDelete={deleteCat} />}
+      <PermissionProvider allowedPermissions={["category_change_permission"]}>
```

Comment:
It's not neccessary to wrap it in PermissionProvider, since the whole page is already wrapped in it (and the permissions are the same)

Reply:
Its true that whole page is wrapped, but permissions are not always the same. Once it is used with `category_add_permission` (from categories/add.tsx) and the other time it is `category_change_permission ` in (categories/edit/[id].tsx)
But it should be probably removed either way.

Reply:
Well, when adding category, the delete button is not displayed anyway (it doesn't make sense, since the object hasn't been created yet)